CHANGELOG.rst

```diff
@@ -0,0 +1,7 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package rcpputils
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.2.0 (2019-02-25)
+------------------
+* Add clang thread safety analysis annotation macros
```

Comment:
IIRC this one is generated by bloom, you don't need to write this manually (?)

Reply:
Haha, ok, I was just trying to copy an existing package very closely, I will delete it

Reply:
It need to be generated prior to bloom, there's catkin_generate_changelog which will fill the content in from the git history. We generally recommend manually revising it after the generation for better readabilty. And then catkin_prepare_release will add the version number and increment your package.xml too. https://index.ros.org/doc/ros2/Tutorials/Releasing-a-ROS-2-package-with-bloom/